configs/gtp-amcts.cfg

```diff
@@ -8,9 +8,19 @@ numBots = 2
 
 maxVisits0 = 600
 searchAlgorithm0 = AMCTS
+useGraphSearch0 = false
 
 maxVisits1 = 32
 searchAlgorithm1 = MCTS
+useGraphSearch1 = true
 
 # This is set to 6 in the original gtp_example.cfg but A-MCTS only supports numSearchThreads = 1
 numSearchThreads = 1
+
+# If we want to keep behavior close to KataGo v1.10, consider setting the
+# following parameters:
+# valueWeightExponent0 = 0.5
+# subtreeValueBiasFactor0 = 0.35
+# subtreeValueBiasWeightExponent0 = 0.8
+# fpuParentWeightByVisitedPolicy0 = false
+# enablePassingHacks0 = false
```

Comment:
I think `fpuParentWeightByVisitedPolicy0` is the important one, not sure if the other ones affect win rate much